### PR TITLE
Directly get the editing column from the database

### PR DESCRIPTION
**Pull request by sbrunner**
fix: #2445

### CODE REVIEW

File: c2cgeoportal/lib/lingua_extractor.py [hunk 1]
```diff
@@ -157,33 +161,22 @@ def _import_layer_wms(self, layer, messages):
...
-            ):
-                if not element.getAttribute("type").startswith("gml:"):
+            exclude = [] if layer.exclude_properties is None else layer.exclude_properties.split(",")
+            last_update_date = self._get_ui_metadata(layer, "lastUpdateDateColumn")
+            if last_update_date is not None:
+                exclude.append(last_update_date)
+            last_update_user = self._get_ui_metadata(layer, "lastUpdateUserColumn")
+            if last_update_user is not None:
+                exclude.append(last_update_user)
+            cls = get_class(str(layer.geo_table, ",".join(exclude)))
```
Comment (gberaudo) [inline c2cgeoportal/lib/lingua_extractor.py]: Remove join

**Comment by gberaudo** [comment]
LGTM

**Comment by gberaudo** [comment]
> def get_class(tablename, session=None, exclude_properties=[]):
@sbrunner, is it really OK to use `[]` static array as default parameter?

**Comment by sbrunner** [comment]
Yes if we don't modify it :-)

**Pull request merged**